src/data/entities/PersistedConfig.ts

```diff
@@ -1,7 +1,11 @@
-import { Permission } from "../../domain/entities/Permission";
+import { Config } from "../../domain/entities/Config";
 
-export interface PersistedConfig {
+export interface PersistedConfig extends Config {
     poeditorToken?: string;
-    settingsPermissions?: Permission;
-    showAllModules?: boolean;
 }
+
+export const defaultConfig = {
+    showAllModules: true,
+    settingsPermissions: {},
```

Comment:
I know it was typed this way before, but we prefer to use `Maybe<Entity>` to explicitly define undefined in order to keep track of it. I tried to search for one comment from Arnau telling me the "more explained reason", but I couldn't find any. Also, I'm one of those who prefer not to define "maybe" just because the data might not be in the datastore. For example in user-extended-app with the AppSettings, we force props (the ones that make sense because they have a value) to be defined and on the repo we handle if the data isn't there. So for example, `showAllModules` in `Config` might be undefined, but is something that I would say that is just `true` or `false` and not a "maybe boolean" state

Reply:
Yes, as a general rule, we prefer `prop: Maybe<T>` than `prop?: T` we need to be explicit on usage. (Note that this is not a mandatory tool, sometimes you defined a function with an options last argument, here it makes sense to omit the ones you are ok with the defaults)

One thing I'd add to line 7 is the type: `export const defaultConfig: Config = {`